test/generators/Generators.scala

```diff
@@ -601,7 +603,8 @@ trait Generators extends SignedInUserGen with ViewModelGenerators {
     for {
       dutyTaxFees <- Gen.listOfN(1, arbitrary[DutyTaxFee])
       classifications <- Gen.listOfN(1, arbitrary[Classification])
-    } yield Commodity(dutyTaxFees = dutyTaxFees, classifications =classifications)
+      transportEquipments <- Gen.listOfN(1, arbitrary[TransportEquipment])
```

Comment:
By only have 1 generated value you have missed bug in the production code.
```suggestion
      transportEquipments <- Gen.listOf(arbitrary[TransportEquipment])
```
```suggestion
      transportEquipments <- varListOf(5, arbitrary[TransportEquipment])
```